src/pages/reservationPage/ReservationPage.jsx

```diff
@@ -1,9 +1,12 @@
 import React, { useState, useEffect } from 'react';
 import { Container, Grid, Paper } from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
 import DateFnsAdapter from '@material-ui/lab/AdapterDateFns';
 import LocalizationProvider from '@material-ui/lab/LocalizationProvider';
-import CircularProgress from '@material-ui/core/CircularProgress';
+import LinearProgress from '@material-ui/core/LinearProgress';
 import Typography from '@material-ui/core/Typography';
+import Snackbar from '@material-ui/core/Snackbar';
+import Slide from '@material-ui/core/Slide';
```

Comment:
The above 4 lines can be reduced into one:
`import { LinearProgress, Typography, Snackbar, Slide} from '@material-ui/core'`